### ISSUE TITLE
 dual screen support on pi4

### ISSUE THREAD

**Issue by dadokkio** [helgeerbe/picframe#98]
Hello,
can picframe be used on raspberry pi 4 on two screen simultaneously? 
For example with 2 instance showing images from different folder on each screen

**Comment by paddywwoof** [comment]
I don't see why two instances of python couldn't be started each pointing to a different screen. I think the processing load on the GPU (and CPU, but that's not significant) would be twice as high as for single instance.

Other people have asked similar questions and I think they have managed but I don't have two micro HDMI plugs and leads to test it.

**Comment by dadokkio** [comment]
I was thinking if that was possible in configuration to keep them running as a service.. but I'll try run them manually to test it

**Comment by paddywwoof** [comment]
Yes, that's what I would try first. ssh into the same Pi with two different sessions and try to start picframe on each but sending output to different monitor. See also conversation here from last year - not sure if relevant now. https://www.raspberrypi.org/forums/viewtopic.php?t=270358

**Comment by helgeerbe** [comment]
Something for the wiki?
